ci/test_cudf_polars_polars_tests.sh

```diff
@@ -24,9 +24,15 @@ rapids-logger "Download wheels"
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 RAPIDS_PY_WHEEL_NAME="cudf_polars_${RAPIDS_PY_CUDA_SUFFIX}" RAPIDS_PY_WHEEL_PURE="1" rapids-download-wheels-from-s3 ./dist
 
+# Download the libcudf built in the previous step
```

Comment:
Let's download libcudf + pylibcudf in one block [like the other script](https://github.com/rapidsai/cudf/blob/a6ca0f0068995e5080e1c8d04410a2a1b9dc8b37/ci/test_wheel_cudf_polars.sh#L27-L29) and also perform one installation step [like the other script](https://github.com/rapidsai/cudf/blob/a6ca0f0068995e5080e1c8d04410a2a1b9dc8b37/ci/test_wheel_cudf_polars.sh#L42-L48).